doc/subversion_and_git_integration.md

```diff
@@ -33,66 +33,86 @@ On that page, enable the "Enable WS for repository management" setting and gener
 for get to save the settings). We need that API key later in our apache config.
 
 Find a place to store the repositories. For this guide we assume that you put your svn repositories in
-/srv/openproject/svn . All things in that repository should be accessible by the apache system user and
-by the user running your openproject server. 
+/srv/openproject/svn and your git repositories in /srv/openproject/git .
+All things in that repository should be accessible by the apache system user and by the user running your openproject server. 
```

Comment:
How about a reference to the configuration.yml here? As there is a dependency, I think it should be mentioned.